sdk/pineconevectordb/arm-pineconevectordb/src/models/models.ts

```diff
@@ -172,11 +172,11 @@ export function offerDetailsDeserializer(item: any): OfferDetails {
 /** User details for an organization */
 export interface UserDetails {
   /** First name of the user */
-  firstName: string;
+  firstName?: string;
```

Comment:
Can you check if this model is being used in any response and response related models ?